src/huggingface_hub/keras_mixin.py

```diff
@@ -530,6 +532,8 @@ def _from_pretrained(
             )
         else:
             storage_folder = model_id
+        if model_kwargs.get("custom_objects"):
```

Comment:
Same here with regards to this being a `model_kwarg` and not a regular kwarg. Also some docstring would help here.